### DISCUSSION TITLE
How to ignore default Chakra UI styling to a specific element?

### DISCUSSION THREAD

**Discussion by jeremiah-blessing** [chakra-ui/chakra-ui#4616] [Q&A]
By default Chakra UI styles heading, paragraph, code, ul, li tags. Is there any way to disable this default behavior?

I am rendering HTML content which consists of a number of these tags that are used to render a blog sort of thing. But the Chakra is removing all the default styles and I don't have any way to interfere with the child nodes of the data to give styles independantly.

It would be much appreciated if any one came across this issue and solved it.

My typical render would be

```
<div> <- I need to disable styling to this div and it's children elements
<p>Some paragraph</p>
<h2>Some heading</h2>
</div>
```

**Comment by niyazidageek** [comment]
You can do it by giving classnames to the areas that you would like to be ignored and later setting the styles up in the CSS. In other words, you will use CSS for resetting the styles that have been set by default. Here is the code:

**CSS:**

```
#someselector {
  all: initial;
}

#someselector * {
  all: unset
}
```

**SCSS:**

```
#someselector {
  all: initial;
  * {
    all: unset;
  }
}
```

After writing this in CSS, you will obtain the wanted result. I hope I could help you!

**Reply by k4l3b4** [reply]
does this work for the body tag?
 i tried but didn't work for me

**Reply by niyazidageek** [reply]
Oh, I am not sure about that. To be honest, I stopped using Chakra UI. It is still raw when it comes to such specific points. Hope you will find a solution!

**Comment by BlackFenix2** [comment]
using the 'initial' and 'unset' just removes all the css styling.

I have a solution. i was trying out an app with a markdown preview but the css reset was messing up with the markdown to html translation.

you can use the "revert" option when wanting to exclude a specific scope of markup to the CSS reset.



CSS:
```
#someselector {
  all: revert;
}

#someselector * {
  all: revert
}
```
SCSS:
```
#someselector {
  all: revert;
  * {
    all: revert;
  }
}
```

**Comment by Abd-essamad-Baaziz** [comment]
I Have the same problem with ListIcon and ListItem
